src/components/Settings/Settings.css

```diff
@@ -38,9 +38,12 @@ body{
 }
 
 .Link a {
-  color : #1DA1F5;
+  color : red;
   text-decoration : none;
 }
+.Link a:hover {
```

Comment:
Use material components from the view library. Don't add in effects like this rather use the buttons provided by the lib.